backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -60,9 +60,9 @@ public TestOrderService(
   }
 
   @Transactional(readOnly = true)
-  public List<TestEvent> getTestResults(String facilityId) {
+  public List<TestOrder> getTestResults(String facilityId) {
     Facility fac = _os.getFacilityInCurrentOrg(UUID.fromString(facilityId));
-    return _terepo.findAllByOrganizationAndFacility(fac.getOrganization(), fac);
+    return _repo.getTestResults(fac.getOrganization(), fac);
```

Comment:
The reason we had the query on TestEvent instead of TestOrder was because we keep expecting to be told to include non-locally-ordered tests in the history page. Admittedly, I'm not sure that it makes sense to have those associated with a facility at all, so there's likely some logic we'll have to build when and if we get there anyway, but that was why it worked this way.